### PR TITLE
Omnisearch: add a link to edit to post title on the results page

### PR DESCRIPTION
**Pull request by jeherve**
Suggested here:
https://wordpress.org/support/topic/omnisearch-add-custom-post-types?replies=14&view=all#post-6865563

### CODE REVIEW

File: modules/omnisearch/omnisearch-posts.php [hunk 1]
```diff
@@ -71,13 +71,18 @@ function prepare_items() {
 	function column_post_title( $post ) {
 		$actions = array();
 		if ( current_user_can( $this->post_type_obj->cap->edit_post, $post ) ) {
+			$post_title = sprintf( '<a href="%s">%s</a>', esc_url( get_edit_post_link( $post->ID ) ), wptexturize( $post->post_title ) );
+		} else {
+			$post_title = wptexturize( $post->post_title );
+		}
+		if ( current_user_can( $this->post_type_obj->cap->edit_post, $post ) ) {
 			$actions['edit'] = sprintf( '<a href="%s">%s</a>', esc_url( get_edit_post_link( $post->ID ) ), esc_html( $this->post_type_obj->labels->edit_item ) );
```
Comment (zinigor) [inline modules/omnisearch/omnisearch-posts.php]: What do you think if we moved this line into the previous `if` since it's the same condtion?

Comment (jeherve) [inline modules/omnisearch/omnisearch-posts.php]: @zinigor That makes sense. I just committed 2ccf7074989a8005c99a4789af225ce42c5ea350, let me know if that does the trick.

**Comment by zinigor** [comment]
Read the changes, tested, looks good, works fine!

**Comment by dereksmart** [comment]
:+1:

**Pull request merged**